dcos/packagemanager.py

```diff
@@ -73,7 +73,7 @@ def has_capability(self, capability):
             raise
         except Exception as e:
             logger.exception(e)
-            return False
+            raise e
```

Comment:
we should catch DCOSHTTPException only and raise those (which would be connecting to the server errors).